cmd/livepeer/livepeer.go

```diff
@@ -302,6 +306,16 @@ func main() {
 				glog.Errorf("Error setting up orchestrator: %v", err)
 				return
 			}
+
+			sigVerifier := pm.NewApprovedSigVerifier(n.Eth)
```

Comment:
I don't think we want to use `ApprovedSigVerifier` right now since it requires ETH calls to check for approved signers and instead we want to use an implementation of `SigVerifier` that only does client side signature verification - there is already a helper function here https://github.com/livepeer/go-livepeer/blob/master/eth/helpers.go#L18

Reply:
@yondonfu so what specific type should we provide to `NewValidator`? - do you mean that I should create a new concrete type that implements the `SigVerifier` interface and implements the `Verify` function from the interface the same way `VerifySig` is implemented in the link you attached?

Reply:
Yeah maybe implement a `DefaultSigVerifier` type that just calls `VerifySig`?

Reply:
Sure, just pushed up a change with that